descriptors/ModuleDescriptor-template.json

```diff
@@ -73,16 +73,7 @@
             "voucher-storage.voucher-number.get",
             "acquisitions-units-storage.units.collection.get",
             "acquisitions-units-storage.memberships.collection.get",
-            "finance.invoice-transaction-summaries.item.post",
```

Comment:
we also should delete dependant interfaces from ModuleDescriptor-template because we don't need them any more
`{
      "id": "finance.invoice-transaction-summaries",
      "version": "2.1"
    },
    {
      "id": "finance.order-transaction-summaries",
      "version": "1.2"
    }`

Reply:
Yes